### ISSUE TITLE
DL3016 does not allow any argument for npm install

### ISSUE THREAD

**Issue by ferrarimarco** [hadolint/hadolint#153]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
`RUN npm install --unsafe-perm` should not raise any error, because my packages are pinned in `package.json`

### Actual behavior

hadolint raises a `DL3016` error.

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:** `Haskell Dockerfile Linter v1.2.6-0-ge03cbb0`

```bash
Dockerfile:53 DL3016 Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`
```
